app/assets/javascripts/github_auth.js.coffee

```diff
@@ -3,13 +3,15 @@ class App.GithubAuth
 
   authenticate: ($githubButton) ->
     $githubButton.click ->
-      github_url_params = [
-        AUTH_URL
-        '?client_id=', $(this).data('clientId')
-        '&redirect_uri=', $(this).data('redirectUri')
-        '&scope=', $(this).data('scope')
-      ]
-      window.location.href = github_url_params.join('')
+      if !$(this).hasClass('clicked')
+        $(this).addClass 'clicked'
+        github_url_params = [
+          AUTH_URL
+          '?client_id=', $(this).data('clientId')
+          '&redirect_uri=', $(this).data('redirectUri')
+          '&scope=', $(this).data('scope')
+        ]
+        window.location.href = github_url_params.join('')
```

Comment:
instead of adding a class and a conditional, why not simply disable the button on click?
```js
$(this).attr('disabled', 'disabled')
```

Reply:
Sure, I will update the code to disable the link after first click.